test/core/image_processing/morphology.cpp

```diff
@@ -0,0 +1,191 @@
+//
+// Copyright 2021 Prathamesh Tagore <prathameshtagore@gmail.com>
+//
+// Use, modification and distribution are subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+//
+
+#include <boost/gil/image_processing/morphology.hpp>
+#include <boost/core/lightweight_test.hpp>
+#include <vector>
+
+namespace gil = boost::gil;
+
+// This function helps us fill pixels of a view given as 2nd argument with elements of the vector
+// given as 1st argument.
+void pixel_fill(std::vector<std::vector<int> > &original_binary_vector, boost::gil::gray8_image_t &original_img)
```

Comment:
- We do C++11, so `(std::vector<std::vector<int> >` should read `(std::vector<std::vector<int>>`
- Attach `&` to the type, not the variable.